styles/main.css

```diff
@@ -38,10 +37,41 @@ header {
   min-height: 8vh;
   align-items: center;
   justify-content: space-between;
+  background-color: white;
 }
 
-.nav-list {
+.mobile-menu {
   display: none;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: white;
+  position: fixed;
+  visibility: hidden;
+}
```

Comment:
- [ ] Your design is almost exceptional team 👏. Here just to make sure we are following the design guideline 100%, Please use the appropriate color for your navbar/hamburger icon. 

### Your design 
![image](https://github.com/ahmedeid6842/Portfolio/assets/69077061/542cafbb-c22c-4984-881d-a5405da3ce86)
### Figma design 
![image](https://github.com/ahmedeid6842/Portfolio/assets/69077061/24fba6a7-2ab2-4bae-a282-6c57679ca0fe)